### PR TITLE
(SIMP-4899) Add MATE support

### PR DESCRIPTION
**Pull request by trevor-vaughan**
* Add MATE support
* Remove 'gdm' default 'dconf' settings since they will break systems
  that do not have 'gdm' installed - BREAKING
* Purge any extra files dropped into dconf db directories

SIMP-4899 #close

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
Need to break out dconf profile additions

**Comment by trevor-vaughan** [comment]
Ready for review

File: Gemfile [hunk 1]
```diff
@@ -18,7 +16,7 @@ group :test do
   gem 'puppet-lint-empty_string-check',   :require => false
   gem 'puppet-lint-trailing_comma-check', :require => false
   gem 'simp-rspec-puppet-facts', ENV.fetch('SIMP_RSPEC_PUPPET_FACTS_VERSION', '~> 2.0')
-  gem 'simp-rake-helpers', ENV.fetch('SIMP_RAKE_HELPERS_VERSION', '~> 4.0')
+  gem 'simp-rake-helpers', ENV.fetch('SIMP_RAKE_HELPERS_VERSION', ['>= 5.2', '< 6.0'])
 end
 
 group :development do
```
Comment (lnemsick-simp) [inline Gemfile]: Latest travis gems no longer work with Ruby 2.1.9

**Comment by lnemsick-simp** [comment]
Are compliance_markup changse needed for this and the gdm module API changes?

**Comment by trevor-vaughan** [comment]
Any comments not addressed in this PR are covered in the following tickets:
* https://simp-project.atlassian.net/browse/SIMP-4935
* https://simp-project.atlassian.net/browse/SIMP-4936
* https://simp-project.atlassian.net/browse/SIMP-4937
* https://simp-project.atlassian.net/browse/SIMP-4938

**Pull request merged**